src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
+    { I18N_CLIENT_GAME_OVER, "LA PARTIE EST FINIE" },
+    { I18N_CLIENT_RENAME, "${1} est maintenant connu comme ${2}" },
+    { I18N_CLIENT_SKIN_CHANGE, "${1} à changer son personnage pour ${2}" },
 
-    { I18N_TNMT_NEW_MAP_LOADED, "new map loaded" },
-    { I18N_TNMT_SPECTATING, "SPECTATING" },
-    { I18N_TNMT_WARMUP, "WARM UP" },
-    { I18N_TNMT_SB_CTC_TITLE, "CATCH THE COIN" },
-    { I18N_TNMT_SB_PLAYER, "PLAYER" },
+    { I18N_TNMT_NEW_MAP_LOADED, "nouvelle carte chargée" },
+    { I18N_TNMT_SPECTATING, "SPECTATEUR" },
+    { I18N_TNMT_WARMUP, "ÉCHAUFFEMENT" },
+    { I18N_TNMT_SB_CTC_TITLE, "ATTRAPEZ LA PIÈCE" },
+    { I18N_TNMT_SB_PLAYER, "JOUEUR" },
     { I18N_TNMT_SB_TOTAL, "TOTAL" },
     { I18N_TNMT_SB_PING, "PING" },
-    { I18N_TNMT_SB_SPECTATORS, "SPECTACTORS:" },
-    { I18N_TNMT_SB_DM_TITLE, "DEATHMATCH" },
+    { I18N_TNMT_SB_SPECTATORS, "SPECTATEURS:" },
+    { I18N_TNMT_SB_DM_TITLE, "MATCH À MORT" },
     { I18N_TNMT_SB_FRAGS, "FRAGS" },
-    { I18N_TNMT_SB_KILLS, "KILLS" },
+    { I18N_TNMT_SB_KILLS, "TUÉS" },
     { I18N_TNMT_SB_SCORE, "SCORE" },
-    { I18N_TNMT_SB_SR_TITLE, "SPEED RACE" },
-    { I18N_TNMT_SB_LAPS, "LAPS" },
-    { I18N_TNMT_SB_BEST, "BEST" },
-    { I18N_TNMT_SB_LAST, "LAST" },
-    { I18N_TNMT_SR_LEAD, "lead:" },
-    { I18N_TNMT_SR_LAP, "lap:" },
-    { I18N_TNMT_SR_BEST, "best:" },
-    { I18N_TNMT_SR_LAST, "last:" },
-    { I18N_TNMT_SR_NA, "N/A" },
-    { I18N_TNMT_PLAYER_JOINS, "${1} joins the game" },
-    { I18N_TNMT_PLAYER_FELL_OFF, "${1} fell off the stage" },
-    { I18N_TNMT_PLAYER_SUICIDE, "${1} committed suicide" },
-    { I18N_TNMT_COIN_RETURNED, "the coin returned to its base" },
-    { I18N_TNMT_COIN_CATCHED, "${1} catched the coin" },
-    { I18N_TNMT_COIN_DROPPED, "the coin dropped" },
-    { I18N_TNMT_RED_FLAG_RETURNED, "the red flag returned to its base" },
-    { I18N_TNMT_BLUE_FLAG_RETURNED, "the blue flag returned to its base" },
-    { I18N_TNMT_RED_FLAG_CATCHED, "${1} catched the red flag" },
-    { I18N_TNMT_BLUE_FLAG_CATCHED, "${1} catched the blue flag" },
-    { I18N_TNMT_RED_FLAG_SAVED, "the red flag returned by ${1}" },
-    { I18N_TNMT_BLUE_FLAG_SAVED, "the blue flag returned by ${1}" },
-    { I18N_TNMT_RED_FLAG_DROPPED, "the red flag dropped" },
-    { I18N_TNMT_BLUE_FLAG_DROPPED, "the blue flag dropped" },
-    { I18N_TNMT_TEAM_RED_SCORED1, "team red scores" },
-    { I18N_TNMT_TEAM_BLUE_SCORED1, "team blue scores" },
-    { I18N_TNMT_TEAM_RED_SCORED2, "TEAM RED SCORES" },
-    { I18N_TNMT_TEAM_BLUE_SCORED2, "TEAM RED SCORES" },
-    { I18N_TNMT_SR_RECORD, "new record of ${1}: ${2}" },
+    { I18N_TNMT_SB_SR_TITLE, "COURSE" },
+    { I18N_TNMT_SB_LAPS, "TOURS" },
+    { I18N_TNMT_SB_BEST, "MEILLEUR" },
+    { I18N_TNMT_SB_LAST, "DERNIER" },
+    { I18N_TNMT_SR_LEAD, "meneur:" },
+    { I18N_TNMT_SR_LAP, "tour:" },
+    { I18N_TNMT_SR_BEST, "meilleur:" },
+    { I18N_TNMT_SR_LAST, "dernier:" },
+    { I18N_TNMT_SR_NA, "" },
```

Comment:
Seems like one can use "ND" in French, but I never saw it used myself so I think "n/a" is understandable enough.
https://fr.wikipedia.org/wiki/N/a